DEVELOPERS.md

```diff
@@ -0,0 +1,29 @@
+# Developers
+
+Notes for developers of the language.
+
+## Build
+
+1. `npm run build`
+  2. Requires files do not contain `\r` (CR) returns and must only include `\n` (LF)
```

Comment:
I assume this is just an issue with the new parser? If so we should just change it to work with either

Reply:
Yep. Since I have a windows machine, I'll work on this then!